docs/reference/changelog.rst

```diff
@@ -0,0 +1,79 @@
+*********
+Changelog
+*********
+
+2.9.0 (2024-Jun-28)
+-------------------
+
+Application
+===========
+
+* Extends ``add_command_groups()`` to accept a sequence instead of a list.
+* Adds support for building architecture-independent artefacts by accepting
+  ``all`` as the ``build-for`` target.
+
+Models
+======
+
+* Adds a default ``Platform`` model. See :doc:`platforms</reference/platforms>`
+  for a reference of the model.
+* Adds a default ``get_build_plan()`` function to the ``BuildPlanner`` class.
+  See :doc:`Build plans</explanation/build-plans>` for an explanation of how
+  the default ``get_build_plan()`` works.
+
+For a complete list of commits, check out the `2.9.0`_ release on GitHub.
+
+2.8.0 (2024-Jun-03)
+-------------------
+
+Commands
+========
+
+* Fixes a bug where the pack command could accept a list of parts as command
+  line arguments.
+* Adds support for commands to accept multiple ``platform`` or ``build-for``
+  values from the command line as comma-separated values.
+
+Remote build
+============
+
+* Retries more API calls to Launchpad.
+* Adds an exponentially backoff to API retries with a maximum total delay of
+  62 seconds.
+* Fixes a bug where the full project name was not used in the remote build log
+  files.
+
+For a complete list of commits, check out the `2.8.0`_ release on GitHub.
+
+2.7.0 (2024-May-08)
+-------------------
+
+Base naming convention
+======================
+
+Applications that use a non-default base naming convention must implement
+``Project._providers_base()`` to translate application-specific base names into
+a Craft Providers base.
+
+The default base naming convention is ``<name>@<channel>``. For example,
```

Comment:
Is it `<name>@<channel>` or `<distro>@<series>`?

Reply:
Good catch, I forgot we agreed to that in craft-platforms.